Bootcamp2015.AmazingRace.Base/Services/DataService.cs

```diff
@@ -0,0 +1,112 @@
+﻿using Bootcamp2015.AmazingRace.Base.Models;
+using Bootcamp2015.AmazingRace.Base.ServiceInterfaces;
+using Microsoft.WindowsAzure.MobileServices;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Bootcamp2015.AmazingRace.Base.Services
+{
+    public class DataService : IDataService
+    {
+        private IMobileServiceClient _serviceClient;
```

Comment:
Good job making this an interface and using dependency injection. It would be good to do some error handling here